scripts/eurostat/health_determinants/bmi/process_test.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to automate the testing for EuroStat BMI process script.
+"""
+
+import os
+from os import path
+import unittest
+import sys
+# module_dir is the path to where this test is running from.
+module_dir = os.path.dirname(__file__)
+sys.path.insert(0, module_dir)
+# pylint: disable=wrong-import-position
+from process import EuroStatBMI
+# pylint: enable=wrong-import-position
```

Comment:
Can you please provide details if pylint is required to be disabled. Also is it possible to import EuroStatBMI without adding to PATH